install_config/configuring_sdn.adoc

```diff
@@ -201,7 +201,7 @@ networkConfig:
 . Add the new port to the iptables rule on each cluster node:
 +
 ----
-# iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 4889 -j ACCEPT
+# iptables -A OS_FIREWALL_ALLOW -p udp -m state --state NEW -m udp --dport 4889 -j ACCEPT
```

Comment:
@bysnupy, based on @bmeng's comment, I think this code block might need to look like this. Will you update your PR? (If you don't have time, I will try to open a new PR later this week.)

----
 # iptables -A OS_FIREWALL_ALLOW -p udp -m state --state NEW -m udp --dport 4789 -j ACCEPT <1>
----
<1> `4789` is the `vxlanPort` value that you set in the master configuration file.

Reply:
@kalexand-rh, thank you for your review, @bmeng's point is appropriated here. We should use the consistent `port`. But this section is `Changing the VXLAN PORT`, and the `vxlan` default `port` is `4789`, so I think we need to change the port in other section as `4889`.

Reply:
Thank you @bysnupy. You're right. :)  I think the format of the codeblock still needs to change to explain the value, though.

Reply:
@kalexand-rh , If you have a suggestion, could you tell me about it please ?

Reply:
I'm confused. I thought the value that you specified in the master config file needed to match the value that you added to your IPtables rule. If that's true, then I think that:
1. At line 197, s/vxlanPort: 4789/vxlanPort: 4889
2. Replace this command:
---
 # iptables -A OS_FIREWALL_ALLOW -p udp -m state --state NEW -m udp --dport 4889 -j ACCEPT <1>
----
<1> `4889` is the `vxlanPort` value that you set in the master configuration file.

If that's not the case, then can you please tell me where you get the `--dport` value in this command?

Reply:
Yeah, the value `vxlanPort` in the config file should match the `--dport` in the iptables command.

Reply:
Thank you for review and suggestion @kalexand-rh @bmeng , I've updated along your suggestion.